packages/workbox-webpack-plugin/package.json

```diff
@@ -31,6 +31,9 @@
     "json-stable-stringify": "^1.0.1",
     "workbox-build": "^3.0.0-alpha.3"
   },
+  "peerDependencies": {
+    "webpack": "^2.0.0 || ^3.0.0"
```

Comment:
This seems unusual to me, is it stipulating that workbox only works with Webpack 2 and 3 (i.e. not 4)?

Reply:
I am following UglifyJS's lead here in limited the acceptable peer dependency: https://github.com/webpack-contrib/uglifyjs-webpack-plugin/blob/025262284c86196d0a4c4fe71e186132579629ec/package.json#L75-L77

(My understanding is that webpack 4 changes the plugin architecture anyway, so I think we should track that separately: https://github.com/GoogleChrome/workbox/issues/1151)